PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutAccountNonce.java

```diff
@@ -198,6 +198,13 @@ public String getPayerId() {
         return payerId;
     }
 
+    public void setPayerInfo(String payerId, String firstName, String lastName, String email) {
```

Comment:
We'll need docs for this since it's public.

Reply:
good callout, I added
```
    /**
     * Sets the payer info if on the nonce object. This is used when the payer information is
     * not returned from the json
     * 
     * @param payerId - The payer id associated with the user
     * @param firstName - The payer's first name
     * @param lastName - The payer's last name
     * @param email - The payer's email
     */
     ```